packages/webpack/test/fixtures/main/node_modules/es6-module-package/index.js

```diff
@@ -1,3 +1,6 @@
+import * as um from 'umd-package';
+
 export const hello = () => {
+  um;
```

Comment:
our newly configured tooling removes unused imports on save

Reply:
alternatively, I think this could work:

```
import 'umd-package'
```